ibis/tests/backends.py

```diff
@@ -528,8 +528,9 @@ class Spark(Backend, RoundHalfToEven):
 
     @staticmethod
     def skip_if_missing_dependencies() -> None:
-        pass
+        pytest.importorskip('pyspark')
 
     @staticmethod
     def connect(data_directory):
-        return Spark.client_testing
+        from ibis.tests.all.conftest import get_spark_testing_client as client
```

Comment:
This can be moved to the top of the file.

Reply:
This results in an import error, looks like it's because of circular imports.